src/planner/update_plan.cpp

```diff
@@ -118,7 +119,7 @@ void UpdatePlan::BuildInitialUpdatePlan(
 UpdatePlan::UpdatePlan(const parser::UpdateStatement *parse_tree)
     : update_primary_key_(false) {
   std::vector<oid_t> column_ids;
-  BuildInitialUpdatePlan(parse_tree, column_ids);
+  BuildInitialUpdatePlan(parse_tree, column_ids, nullptr);
```

Comment:
I already spent some time to fix this, and have fixes in my local machine.  I will upload them as a separate PR, today.